src/frontend/e2e/tests/filters.spec.ts

```diff
@@ -0,0 +1,48 @@
+import {test,expect} from '@playwright/test'
+import {LoginPage} from '../page-objects/LoginPage'
+import {SamplesPage} from '../page-objects/SamplesPage'
+
+test.describe("Filter Test Suite",()=>{
+    let loginPage: LoginPage;
+    let samplePage: SamplesPage;
+
+    test.beforeEach(async ({ page })=>{
+        await page.setViewportSize({ width: 1600, height: 1200 });
+        loginPage = new LoginPage(page);
+        samplePage = new SamplesPage(page);
+        await loginPage.login('lbrambila@contractor.chanzuckerberg.com','Br@mb1la');
+    })
+
+    test("Genome Recovery filter by [complete] test",async ()=>{
+       await samplePage.filterGenomeRecovery('Complete');
+       const statusList = await samplePage.getSampleStatusList();
+       await expect(samplePage.sampleStatusesList).toHaveText(statusList);
+
+    })
+
+    test.skip("Genome Recovery filter by [failed] test",async ()=>{
```

Comment:
Why skip

Reply:
OK, we don't have samples that have failed. Removing test and adding when we have test data